build.gradle

```diff
@@ -44,7 +41,6 @@ allprojects {
 
 subprojects {
 	apply plugin: 'signing'
-	apply plugin: 'io.spring.javaformat'
```

Comment:
This and some other `apply plugin:` lines removed because the plugins will be applied to this and all subprojects when declared in the `plugins` block, unless `apply false` is specified. For the javaformat plugin, that does change the behavior from applying the plugin on only subprojects to applying to all projects. That doesn't cause any issue in this repo, but we'll have to check if it causes an issue elsewhere.